### ISSUE TITLE
Error in configure method

### ISSUE THREAD

**Issue by mcorbelli** [jamauro/method#3]
Configs passed to the "configure" method, are not applied to all methods

In main.js in server folder

```javascript
const hello = () => { console.log('hello') }
const there = () => { console.log('there') }
const world = () => { console.log('world') }

Methods.configure({
    before: [hello, there], // global before function(s) that will run before all methods
    after: world, // global after function(s) that will run after all methods
    serverOnly: true, // globally make all methods serverOnly, aka disable Optimistic UI, by setting to true
    options: {
        returnStubValue: true, // make it possible to get the ID of an inserted item on the client before the server finishes
        throwStubExceptions: true,  // don't call the server method if the client stub throws an error, so that we don't end up doing validations twice
    },
    arePublic: false, // by default all methods will be protected by authentication, override it for all methods by setting this to true
    basePath: `/imports/methods`, // used when dynamically importing methods
    loggedOutError: new NotLoggedIn(), // customize the logged out error
});
``` 

In imports/methods/account.methods.js
```javascript
export const editEmail = createMethod({
    name: AccountMethods.EDIT_EMAIL,
    validate: new SimpleSchema().validator(),
    run() {
        throw new UnimplementedMethod();
    }
});
```

**Comment by jamauro** [comment]
Hey @mcorbelli, put `Methods.configure({...})` in a file that is loaded on both the client and the server and then you should be good to go. I'm assuming that your `/imports/methods` are already being loaded on both the client and server. If you're trying to do something different, let me know and show me how you're invoking the method.

Also note that everything in `Methods.configure` is an optional override so you only need to put the properties you want to change in it. In your case, it would look like this:

```js
import { Methods } from 'meteor/jam:method';

Methods.configure({
    serverOnly: true,
    basePath: `/imports/methods`, 
    loggedOutError: new NotLoggedIn(),
});
```

**Comment by jamauro** [comment]
Hey @mcorbelli, I'm assuming you were able to get things squared away. If you're still running into issues, please open a new issue. Thanks!
